shared/chat/manage-channels/container.js

```diff
@@ -64,9 +68,11 @@ const mapDispatchToProps = (dispatch: Dispatch, {navigateUp, routePath, routePro
       )
       dispatch(saveChannelMembership(teamname, channelsToChange))
     },
-    onPreview: (conversationIDKey: string) => {
+    onPreview: (conversationIDKey: string, previousPath?: string[]) => {
```

Comment:
this should be the `Path` type

Reply:
`navigateTo` takes `PropsPath<*>`, but `pathSelector` returns `I.List<string>`, so that's what this actually is (I retyped in latest commits)

Reply:
though I guess you could argue either way?